app/code/Magento/CatalogInventoryGraphQl/Model/Resolver/StockStatusProvider.php

```diff
@@ -48,6 +49,20 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         $stockStatus = $this->stockStatusRepository->get($product->getId());
         $productStockStatus = (int)$stockStatus->getStockStatus();
 
-        return $productStockStatus === StockStatusInterface::STATUS_IN_STOCK ? 'IN_STOCK' : 'OUT_OF_STOCK';
```

Comment:
You removed that line, but there's no value for it when product is not downloadable. So the `return` statement will be always false.